pkgs/development/python-modules/plyer/default.nix

```
@@ -0,0 +1,77 @@
+{ lib, buildPythonPackage, fetchFromGitHub, mock, python3Packages, python, stdenv }:
```

```suggestion
{ lib, buildPythonPackage, fetchFromGitHub, mock, python, stdenv }:
```
this is not allowed in python packages. Please list the packages individually.